ios/RNSScreenStack.mm

```diff
@@ -272,18 +270,10 @@ - (void)didMoveToWindow
 - (void)maybeAddToParentAndUpdateContainer
 {
   BOOL wasScreenMounted = _controller.parentViewController != nil;
-#ifdef RCT_NEW_ARCH_ENABLED
   BOOL isScreenReadyForShowing = self.window;
```

Comment:
I don't know if keeping the definition of `isScreenReadyForShowing = self.window` is okay. I mean, by definition when the self.window is not null, this means that the screen has been mounted, so we can show it in the hierarchy. Maybe checking if self.window is not nil could be better in the `if`s below?

Reply:
https://github.com/software-mansion/react-native-screens/pull/2188/commits/6a46af749d3d42369b08122f7a667ad4430aa2c0